Library/Include/Babylon/Console.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
```

Comment:
I'd like to see a separation between the BabylonNative trunk and the plugins. Maybe add a plugin folder and a subdirectory per plugin?

Reply:
I agree that that's the direction I want it to go, but I'm not sure I want to make that a part of this PR. Depending on how many of these plugins we expect there to be, I kind of feel like there should be some kind scalable story about how to cleanly add them to a project, and I don't really have a good feel for what that should look at right now. In short, I totally agree something like this should be a thing, but I don't think it should be a part of this PR.

Reply:
File an issue after merging then?

Reply:
#119